network/network_windows.go

```diff
@@ -16,6 +15,7 @@ import (
 	"github.com/Microsoft/hcsshim"
 	"github.com/Microsoft/hcsshim/hcn"
 	"github.com/google/uuid"
+	"github.com/pkg/errors"
```

Comment:
Assuming this is intentionally added and not added by the formatting commands.

Reply:
we need to use the formatted errors, so it's intentionally added here